reporter/src/main/kotlin/utils/FreemarkerTemplateProcessor.kt

```diff
@@ -199,6 +199,11 @@ class FreemarkerTemplateProcessor(
          */
         val licenseFiles: ResolvedLicenseFileInfo by lazy { input.licenseInfoResolver.resolveLicenseFiles(id) }
 
+        /**
+         * A name of this package that can be referenced from templates. It is generated from the identifier.
+         */
+        val name: String = "${id.namespacePrefix()}${id.name}:${id.version}"
```

Comment:
commit: If this is just about eliminating code duplication, why not provide a helper function which can be called from the template? e.g. this way the package name would remain similarly customizable from the template.

Reply:
This commit was dropped. The human-readable package name is now generated within the template.